terms/correlation_intractability.md

```diff
@@ -0,0 +1,15 @@
+# Correlation-Intractability (CI)
+
+Let $R$ denote some property of pairs $(y,h(y))$. A hash family $H$ satisfies **CI** for $R$ if it is computationally infeasible to
+find a pair $(y, h(y))$ satisfying property $R$.
+
+Suppose $I$ is an [IP](ip.md) for a [language](language.md) $L$ such that $I$ satisfies
+[round-by-round soundness](../articles/proofs-arguments-and-zero-knowledge/chapter_5.md#Round-by-round%20soundness%20requirements).\
```

Comment:
where is the round-by-round soundness requirement.md?

Reply:
ok sure I'll add this file soon.